src/js/providers/html5.js

```diff
@@ -59,8 +59,15 @@ define([
         // Are we buffering due to seek, or due to playback?
         this.seeking = false;
 
-        var _dispatcher = new eventdispatcher('provider.' + _name);
-        _.extend(this, _dispatcher);
+        _.extend(this, Events);
+
+        // Overwrite the event dispatchers to block on certain occasions
+        this.sendEvent = function(type, data) {
+            if (!_attached) {
+                return;
+            }
+            this.trigger(type, data);
+        };
```

Comment:
Let's not use `sendEvent` anymore since it can be confusing if both `sendEvent` and `trigger` can be used on an object. You can do:
```
this.trigger = function(type, args) {
    if (!_attached) {
        return;
    }
    return Events.trigger.call(this, type, args);
};
```